tools/c7n_gcp/c7n_gcp/resources/compute.py

```diff
@@ -599,3 +599,51 @@ def get_resource_params(self, model, resource):
                   }}
 
         return result
+
+
+@resources.register('gce-project')
+class Project(QueryResourceManager):
+    """GCP resource: https://cloud.google.com/compute/docs/reference/rest/v1/projects"""
+    class resource_type(TypeInfo):
+        service = 'compute'
+        version = 'v1'
+        component = 'projects'
+        enum_spec = ('get', '[@]', None)
+        name = id = 'name'
+        default_report_fields = ["name"]
+        asset_type = 'compute.googleapis.com/Project'
+
+        @staticmethod
+        def get(client, resource_info):
+            return client.execute_command(
+                'get', {'project': resource_info['project_id']})
+
+
+@resources.register('zone')
+class GCPZones(QueryResourceManager):
+    """GCP resource: https://cloud.google.com/compute/docs/reference/rest/v1/zones"""
+    class resource_type(TypeInfo):
+        service = 'compute'
+        version = 'v1'
+        component = 'zones'
+        enum_spec = ('list', 'items[]', None)
+        scope = 'project'
+        name = id = 'name'
+        default_report_fields = ['id', 'name', 'dnsName', 'creationTime', 'visibility']
+        asset_type = "compute.googleapis.com/compute"
+        scc_type = "google.cloud.dns.ManagedZone"
```

Comment:
this resource is mixing up two different things.

a compute zone, which is a location that provides compute within a region
and dns zones, which are for holding dns records (Ala a domain).

they are separate resource types

Reply:
aknowledged. Fixed